index.html

```diff
@@ -180,20 +182,20 @@ <h3 class="bannerbox__title">I need a ride</h3>
 
                             <div class="form-column">
                                 <div class="form-group">
-                                    <label for="rideArea">Collection ZIP code</label>
+                                    <label for="rideArea">Pick up ZIP code</label>
                                     <input type="text" class="form-input" pattern="(^\d{5}$)|(^\d{5}-\d{4}$)" id="rideArea" placeholder="Where you can meet the driver" name="RiderCollectionZIP" required>
                                     <div class="help-block with-errors"></div>
                                 </div>
                                 <div class="form-group">
-                                    <label for="rideDestinationZIP">Destination ZIP code</label>
+                                    <label for="rideDestinationZIP">To where do you need a ride?</label>
```

Comment:
I think this text is supposed to be for the input placeholder, not the label text.